src/components/Accounts/Actions/FeeFineActions.js

```diff
@@ -28,7 +28,7 @@ import {
   calculateSelectedAmount,
   loadServicePoints,
 } from '../accountFunctions';
-import { localizeCurrencyAmount } from '../../util/localizeCurrencyAmount';
+import localizeCurrencyAmount from '../../util/localizeCurrencyAmount';
```

Comment:
after updating src/hooks/index.js file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import